Sources/Valet/SinglePromptSecureEnclaveValet.swift

```diff
@@ -18,14 +18,14 @@
 //  limitations under the License.
 //
 
-#if os(iOS) || os(macOS)
+#if canImport(LocalAuthentication)
```

Comment:
Now that we're no longer supporting Xcode 9, we can use `canImport` to enable `SinglePromptSecureEnclaveValet` on tvOS 10+.

Reply:
Do we still need the `@available(tvOS 10.0, *)` given the `canImport` here?

Reply:
Yup. If I get rid of it we get build failures 🙂. Here's my understanding as to why:

1.`#if canImport` is a compiler flag that enables a class for compilation in the case where the SDK has access to the library in question.
1. `@available` is a SDK flag that makes a class as available on particular OS versions.

As such, when we're targeting a `tvOS` SDK on Xcode 11, `LocalAuthentication` can always be imported because Xcode 11's SDK has `LocalAuthentication` on `tvOS`. We still need to limit the availability of the class that is now bundled in our `tvOS` library to ensure there are no crashes when running this code on tvOS 9.